_includes/scripts.html

```diff
@@ -10,4 +10,15 @@
     anchors.add()
 </script>
 
+<script type="text/javascript">
+$('.buttonGroup button').on('click', function(filterContent) {
+    $('.buttonGroup button').attr('disabled', false);
+    $(this).attr('disabled', true);
+    var tag = $(this).text();
+    $('.news').hide();
+    $('.' + $(this).attr('id')).fadeIn();
+    filterContent.preventDefault();
+})
+</script>
```

Comment:
I would recommend a somewhat more specific class name for this purpose, since this script is universal to the site. It will also help further distinguish it from Bootstrap's own `btn-group` class.
```suggestion
<script type="text/javascript">
$('.filterBtnGroup button').on('click', function(filterContent) {
    $('.filterBtnGroup button').attr('disabled', false);
    $(this).attr('disabled', true);
    var tag = $(this).text();
    $('.news').hide();
    $('.' + $(this).attr('id')).fadeIn();
    filterContent.preventDefault();
})
</script>
```